modules/files/check-link-targets.sh

```diff
@@ -30,6 +30,9 @@ for sourcePath in "$@" ; do
     if cmp -s "$sourcePath" "$targetPath"; then
       # First compare the files' content. If they're equal, we're fine.
       warnEcho "Existing file '$targetPath' is in the way of '$sourcePath', will be skipped since they are the same"
+    elif [[ ! -L "$targetPath" && -n "$HOME_MANAGER_BACKUP_COMMAND" ]] ; then
+      # Next, try to run the custom backup command. Assume this always succeeds.
```

Comment:
maybe add a `verboseEcho` to add some logging that we've hit this block if we're creating a condition for it.